Reading List/Reading List/ReadingListTableViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  ReadingListTableViewController.swift
+//  Reading List
+//
+//  Created by Keri Levesque on 1/28/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ReadingListTableViewController: UITableViewController {
+
+    let bookController = BookController()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        switch section {
+                case 0: return bookController.unreadBooks.count
+                default: return bookController.readBooks.count
+                }
+    }
+
+    private func bookFor(indexPath: IndexPath) -> Book {
+        if indexPath.section == 0 {
+                  return bookController.readBooks[indexPath.row]
+              } else {
+                  return bookController.unreadBooks[indexPath.row]
+              }
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "CellSegue", for: indexPath)
```

Comment:
With a custom cell we always need to unwrap the cell and then downcast this line as the TableViewCell file we have in our code. Like this:
`guard let cell = tableView.dequeueReusableCell(withIdentifier: "CellSegue", for: indexPath) as? BookTableViewCell else { return UITableViewCell() }`